core/trino-spi/src/main/java/io/trino/spi/expression/StandardFunctions.java

```diff
@@ -81,7 +81,7 @@ private StandardFunctions() {}
      */
     public static final FunctionName NEGATE_FUNCTION_NAME = new FunctionName("$negate");
 
-    public static final FunctionName LIKE_PATTERN_FUNCTION_NAME = new FunctionName("$like_pattern");
```

Comment:
The old name is what we use internally: `io.trino.type.LikeFunctions#LIKE_PATTERN_FUNCTION_NAME`

I think the internal name should be updated accordingly.